From 83d10cccd5de617cdabfc5b9604a210d8d603329 Mon Sep 17 00:00:00 2001
From: Bret Ambrose <bretambrose@gmail.com>
Date: Tue, 31 Aug 2021 13:58:03 -0700
Subject: [PATCH] Remove unneeded include

---
 tests/sigv4_signing_tests.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/sigv4_signing_tests.c b/tests/sigv4_signing_tests.c
index 4638671..fef3bf2 100644
--- a/tests/sigv4_signing_tests.c
+++ b/tests/sigv4_signing_tests.c
@@ -18,7 +18,6 @@
 #include <aws/common/string.h>
 #include <aws/http/request_response.h>
 #include <aws/io/file_utils.h>
-#include <aws/io/pki_utils.h>
 #include <aws/io/stream.h>
 #include <aws/io/uri.h>
 
